bibtex-actions.el

```diff
@@ -161,7 +161,9 @@ means no action."
   :type '(choice (const :tag "Prompt" 'prompt)
                  (const :tag "Ignore" nil)))
 
-(defcustom bibtex-actions-embark-dwim t
+(defvar bibtex-actions-at-point-fallback t)
```

Comment:
Make the fallback variable a defcustom too?

Reply:
Funny; I was just writing that question.

Reply:
Do you want me to credit you on the commit, BTW?

I know it's a small suggestion, but figure I may as well ask.

Reply:
No need, it is just a small suggestion.

Reply:
There's already a defcustom for this. Do you see any issue with this?

``` elisp
(defcustom bibtex-actions-at-point-fallback 'prompt
  "Fallback action for `bibtex-actions-at-point'.
The action is used when no citation key is found at point.
`prompt' means choosing entries via `bibtex-actions-read' and nil
means no action."
  :group 'bibtex-actions
  :type '(choice (const :tag "Prompt" 'prompt)
                 (const :tag "Ignore" nil)))
```

Reply:
Ah okay right. Okay to keep it like this I guess.

Reply:
Hmm ... I'm not understanding why it's not doing the fallback prompting, even in your version with the boolean.

Reply:
Oh no idea, I am sorry - I didn't test this. I just wrote this quickly to illustrate the simplification I thought is useful. Something along these lines should work of course.